aesfield/field.py

```diff
@@ -23,50 +22,46 @@ def __init__(self, *args, **kwargs):
         self.aes_key = kwargs.pop('aes_key', '')
         super(AESField, self).__init__(*args, **kwargs)
 
+    def deconstruct(self):
+        name, path, args, kwargs = super(HandField, self).deconstruct()
+        # TODO: Should `aes_method` be present in the serialized form too?
```

Comment:
Any chance of getting an answer to this before merging the pull?  I'm loathed to add more TODOs any any code.